ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/NetworkRunner.java

```diff
@@ -89,7 +84,7 @@ public void start() {
     if (started.compareAndSet(false, true)) {
       LOG.info("Starting Network.");
       setupHandlers();
-      networkExecutor.submit(network);
```

Comment:
Not entirely sure why this ever had its own dedicated executor - starting the network is actually very quick as it basically just opens a port and returns.